README.md

```diff
@@ -99,6 +99,123 @@ Run the following command:
 docker-compose up -d
 ```
 
+#### Docker Swarm w/ Docker Secrets:
+
+This section discusses running an instance of ibeam inside a locked docker swarm, and using the docker swarm facilities for managing secrets.
+Please refer to the following articles for in-depth details on docker swarm, locking the swarm, and using docker secrets.
+
+- [Swarm mode overview](https://docs.docker.com/engine/swarm/)
+- [Lock your swarm to protect its encryption key](https://docs.docker.com/engine/swarm/swarm_manager_locking/)
+- [Manage sensitive data with Docker secrets](https://docs.docker.com/engine/swarm/secrets/)
+
+Once you have a locked docker swarm instance initialized, you can create the secrets.
+On your host system create two secure (meaning not world-readable) files containing your Interactive Brokers account name and password:
+
+1. ib.account.txt
+2. ib.password.txt
+
+Next, inject these secrets into the docker swarm by using `docker secret create`:
+
+```posh
+docker secret create IBEAM_ACCOUNT_v1 ib.account.txt
+docker secret create IBEAM_PASSWORD_v1 ib.password.txt
+```
+
+Once you've initialized the secrets you should remove the original files from your host system.
+
+Create a `docker-compose.yml` file that references the new secrets, and that configures its environment to pull the secrets from the container's filesystem:
+
+```yaml
+version: "3.7"
+
+secrets:
+  IBEAM_ACCOUNT_v1:
+    external: true
+  IBEAM_PASSWORD_v1:
+    external: true
+
+services:
+  ibeam:
+    image: "voyz/ibeam"
+    environment:
+      IBEAM_SECRETS_SOURCE: "fs"
+      IBEAM_ACCOUNT: "/run/secrets/IBEAM_ACCOUNT"
+      IBEAM_PASSWORD: "/run/secrets/IBEAM_PASSWORD"
+    secrets:
+      - source: "IBEAM_ACCOUNT_v1"
+        target: "/run/secrets/IBEAM_ACCOUNT"
+        uid: "1000"
+        gid: "1000"
+        mode: 0400
+      - source: "IBEAM_PASSWORD_v1"
+        target: "/run/secrets/IBEAM_PASSWORD"
+        uid: "1000"
+        gid: "1000"
+        mode: 0400
+    volumes:
+      - type: "bind"
+        source: "conf.yaml"
+        target: "/srv/clientportal.gw/root/conf.yaml"
```

Comment:
Doesn't this conflict with the way IBeam handles using the `conf.yaml` through the Inputs Directory? See https://github.com/Voyz/ibeam/wiki/Inputs-And-Outputs#-inputs-directory

Reply:
You're correct.  I'll rework this to discuss conf.yaml in the context using an inputs directory.